### PR TITLE
Make all Valence Builders Chain Agnostic

### PR DESCRIPTION
**Pull request by lexzaiello**
Small change for the sake of API consistency: add support for specifying custom chains in valence contract builders.

### CODE REVIEW

File: src/utils/setup/valence.rs [hunk 1]
```diff
@@ -392,14 +457,16 @@ impl TestContext {
...
         min_auction_amount: impl AsRef<[(&'a str, MinAmount)]>,
         server_addr: impl AsRef<str>,
     ) -> Result<(), Error> {
         let mut contract_a: CosmWasm = self
             .get_contract()
             .contract(AUCTIONS_MANAGER_CONTRACT_NAME)
+            .src(chain)
             .get_cw();
-        let neutron = self.get_chain(NEUTRON_CHAIN_NAME);
+        let neutron = self.get_chain(chain);
```
Comment (keyleu) [inline src/utils/setup/valence.rs]: if it's chain agnostic let's change the name of the variable here.

File: src/utils/setup/valence.rs [hunk 2]
```diff
@@ -620,14 +698,15 @@ impl TestContext {
...
         }
     }
 
-    fn tx_update_auction_oracle(&mut self, sender_key: &str) -> Result<(), Error> {
+    fn tx_update_auction_oracle(&mut self, sender_key: &str, chain: &str) -> Result<(), Error> {
         // The auctions manager for this deployment
-        let contract_a = self.get_auctions_manager().get_cw();
-        let neutron = self.get_chain(NEUTRON_CHAIN_NAME);
+        let contract_a = self.get_auctions_manager().src(chain).get_cw();
+        let neutron = self.get_chain(chain);
```
Comment (keyleu) [inline src/utils/setup/valence.rs]: same as above

File: src/utils/setup/valence.rs [hunk 3]
```diff
@@ -758,11 +842,12 @@ impl TestContext {
...
         &mut self,
         sender_key: &str,
+        chain: &str,
         end_blocks: u128,
         pair: (TDenomA, TDenomB),
     ) -> Result<(), Error> {
-        let manager = self.get_auctions_manager().get_cw();
-        let neutron = self.get_chain(NEUTRON_CHAIN_NAME);
+        let manager = self.get_auctions_manager().src(chain).get_cw();
+        let neutron = self.get_chain(chain);
```
Comment (keyleu) [inline src/utils/setup/valence.rs]: same as above

**Review by keyleu — CHANGES_REQUESTED** [review]

File: src/utils/setup/valence.rs (hunk 1 above)
Comment (lexzaiello) [reply src/utils/setup/valence.rs]: Ok! Fixed.

File: src/utils/setup/valence.rs (hunk 2 above)
Comment (lexzaiello) [reply src/utils/setup/valence.rs]: Fixed!

File: src/utils/setup/valence.rs (hunk 3 above)
Comment (lexzaiello) [reply src/utils/setup/valence.rs]: Done.

**Review by keyleu — APPROVED** [review]

**Pull request merged**